docs/work_tools/all.devops_docker.how_to_guide.md

```diff
@@ -332,15 +329,8 @@
 - The command above will generate `./tmp.requirements.txt` with the list of the
   imported packages, e.g.,
   ```markdown
-  amp==1.1.4
-  async_solipsism==0.3
-  beautifulsoup4==4.11.1
-  botocore==1.24.37
-  cvxopt==1.3.0
-  cvxpy==1.2.0
-  dill==0.3.4
-  environs==9.5.0
-  ...
+  amp==1.1.4 async_solipsism==0.3 beautifulsoup4==4.11.1 botocore==1.24.37
```

Comment:
Same.

Let's review the PR before sending it out

Reply:
Reverted.